internal/pkg/partition/constants.go

```diff
@@ -25,6 +25,7 @@ const (
 	FilesystemTypeZeroes FileSystemType = "zeroes"
 	FilesystemTypeXFS    FileSystemType = "xfs"
 	FilesystemTypeVFAT   FileSystemType = "vfat"
+	FileSystemTypeExt4   FileSystemType = "ext4"
```

Comment:
maybe split this into a different PR :thinking:

Reply:
should be good, it's only used in the installer for now